src/index.js

```diff
@@ -1 +1,23 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import table from './table.js';
+import { getScores, addScore } from './api.js';
+
+const btn = document.querySelector('.submit-btn');
+
+async function render() {
+  const array = await getScores();
+  table(array);
+}
+render();
```

Comment:
- [ ] Ensure to use of arrow functions instead of the `function` keyword as this is a project requirement to use ES6 syntax.